README.md

```diff
@@ -23,6 +23,8 @@ are welcome.  See [HACKING] for more info.
 - [ ] re-use strong UX patterns from existing console mailers like mutt and sup
 - [ ] extensible with plug-ins
 - [x] enforce sane configuration by using Haskell's type system
+- [ ] enforce security against malicious emails by using more expressive types
```

Comment:
Didn't tick it yet, since I thought there is more to come (I had your recent commits of the tainting of data in mind)? What do you think?

Reply:
Sure.  I just think maybe we want to move the whole section a bit further down in the README, because they look like huge gaps, even if we got to 90% of where we want to be...

Reply:
Hm... maybe it's not the right format then? Perhaps changing it into growing feature list which puts us apart and points out the motivation as to why we're implementing a new mailer would be better?

Reply:
I dunno, I'll just merge this, we can re-org later.